packages/camera/camera_avfoundation/example/integration_test/camera_test.dart

```diff
@@ -81,8 +81,15 @@ void main() {
       bool previousPresetExactlySupported = true;
       for (final MapEntry<ResolutionPreset, Size> preset
           in presetExpectedSizes.entries) {
-        final CameraController controller =
-            CameraController(cameraDescription, preset.key);
+        final CameraController controller = CameraController.withSettings(
+          cameraDescription,
+          mediaSettings: MediaSettings(
+            resolutionPreset: preset.key,
+            fps: 15,
+            videoBitrate: 200000,
+            audioBitrate: 32000,
```

Comment:
Why do all of these calls in the integration tests in every package need seemingly arbitrary values added to every call, which aren't actually being used in any testing assertions. Aren't these optional?